app/Testing/Context/Eloquent/Users.php

```diff
@@ -29,6 +29,7 @@ public function thereIsAnUserCreated(): void
     /**
      * @When there is an user created:
      * @When there are users created:
+     * @Given there is/are already existing user(s)   
```

Comment:
Behat is ignoring keywords Given/When/Then, so you could use `Given there is an user created:` in your test.